tests/test_repository_lib.py

```diff
@@ -86,24 +81,25 @@ def tearDownClass(cls):
     tuf.roledb.clear_roledb(clear_all=True)
     tuf.keydb.clear_keydb(clear_all=True)
 
-    shutil.rmtree(cls.temporary_directory)
-
 
   def setUp(self):
+    # We are inheriting from custom class.
```

Comment:
I don't think we need the comment to justify chaining up to the parent implementation of the method here, or throughout the tests you have edited.